dapps/W3MWagmi/.gitignore

```diff
@@ -69,9 +68,14 @@ yarn-error.log
 sentry.properties
 
 # Others
-.env
+.env.*
+*.env
+!.env.example
 
 # Expo
 .expo
 dist/
-web-build/
\ No newline at end of file
+web-build/
+
+# Secrets
+secrets.properties
```

Comment:
Do we need this gitognore as well? It looks like these files can be added to the root ignore file